Parser fails for template with inline SVG containing a viewBox attribute

**Tell us about your environment**

* **ESLint Version:** 4.0.0
* **eslint-plugin-vue Version:** 3.1.3
* **Node Version:** 8.0.0

**Please show your full configuration:**
Vanilla

**What did you do? Please include the actual source code causing the issue.**

<!-- Paste the source code below: -->
```vue
<template>
  <div>
    <svg viewBox="0 0 40 40"></svg>
  </div>
</template>
```

**What did you expect to happen?**
No error reported.

**What actually happened? Please include the actual, raw output from ESLint.**

  0:0  error  Parsing error: Cannot read property 'startOffset' of undefined


https://github.com/octref/vetur/issues/291

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "requireindex": "^1.1.0",
-    "vue-eslint-parser": "^1.1.0-6"
+    "vue-eslint-parser": "^1.1.0-7"
   },
   "devDependencies": {
     "@types/node": "^4.2.6",

--- a/tests/lib/rules/no-parsing-error.js
+++ b/tests/lib/rules/no-parsing-error.js
@@ -34,6 +34,18 @@ tester.run('no-parsing-error', rule, {
     {
       filename: 'test.vue',
       code: '<template>{{a + b + c}}</template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><svg class="icon"><use xlink:href="#chevron"></use></svg></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><svg viewBox="0 0 40 40"></svg></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<table><tr><td></td></tr></table>'
     }
   ],
   invalid: [

